cmd/login/auth.go

```diff
@@ -76,19 +76,19 @@ func handleAuth(ctx context.Context, out io.Writer, i *installation.Installation
 			connectorID = customerConnectorID
 		}
 	}
+
 	authURL := auther.GetAuthURL(connectorID)
-	{
-		// Open the authorization url in the user's browser, which will eventually
-		// redirect the user to the local web server we'll create next.
-		err = open.Run(authURL)
-		if err != nil {
-			return oidc.UserInfo{}, microerror.Mask(err)
-		}
-	}
 
 	fmt.Fprintf(out, "\n%s\n", color.YellowString("Your browser should now be opening this URL:"))
 	fmt.Fprintf(out, "%s\n\n", authURL)
 
+	// Open the authorization url in the user's browser, which will eventually
+	// redirect the user to the local web server we'll create next.
+	err = open.Run(authURL)
+	if err != nil {
+		fmt.Fprintf(errOut, "%s\n\n", color.RedString("Couldn't open the default browser. Please access the URL above to continue logging in."))
```

Comment:
Feels like it should be yellow as you can recover.

Reply:
I don't think we have a strong definition regarding what's allowed to go to stderr. In this case I think either stdout or stderr is fine.